5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.driver.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import TodoListItem from "./TodoListItem";
+import { BaseDriver } from "../../../test/baseDriver";
+import dataHooks from "../../../dataHooks/dataHooks";
+import { mockTodo, throwErrorOnMissingWrapper } from "../../../test/utils";
+import { Item } from "../../../interfaces/interfaces";
+
+export class TodoListItemDriver extends BaseDriver {
+  private todoItem: Item = mockTodo();
+  private removeTodo: jest.Mock = jest.fn();
+
+  private buttonClickHandler = (dataHook: string) => {
+    if (!this.wrapper) return throwErrorOnMissingWrapper();
+    const buttonElem = this.wrapper.getByTestId(dataHook);
+    fireEvent.click(buttonElem);
+    return buttonElem;
```

Comment:
why are you returning it? you wouldn't want to return the button item to the test but be more specific with the api you're exposing

Reply:
Yes, my bad, I don't need to return it